alpine/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ target_link_libraries (PenningTrap ${IPPL_LIBS})
 add_executable (LandauDamping LandauDamping.cpp)
 target_link_libraries (LandauDamping ${IPPL_LIBS})
 
+add_executable (FreeElectronLaser FreeElectronLaser.cpp)
```

Comment:
As mentioned in the meeting, maybe move the Free Electron Laser to a separate directory outside of alpine.